index.js

```diff
@@ -223,6 +235,7 @@ class PhotoView {
   reset() {
     this.instances.forEach(photoViewInstance => {
       photoViewInstance.reset();
+      photoViewInstance.resetScale();
```

Comment:
this one is not required as the **resetScale** is already called from the **reset** method itself.